mech/__main__.py

```diff
@@ -185,16 +240,48 @@ def mech_pause():
     vm.pause()
 
 
-def mech_ssh(user="mech"):
+def mech_ssh(user):
     vm = get_vm()
     ip = vm.ip()
+    if user == None:
+        user = get_vm_user()
     if ip:
-        puts(colored.green(ip))
+        puts("Connecting to {}".format(colored.green(ip)))
         os.system('ssh {}@{}'.format(user, ip))
     else:
         puts(colored.red("IP not found"))
         return
 
+def mech_scp(user, src, dst):
+    vm = get_vm()
+    ip = vm.ip()
+    if user == None:
+        user = get_vm_user()
+    if ip:
+        src_is_host = False
+        if src[0] == ":":
+            src = src[1:]
+            src_is_host = True
+
+        dst_is_host = False
+        if dst[0] == ":":
+            dst = dst[1:]
+            dst_is_host = True
+
+        if src_is_host and dst_is_host:
```

Comment:
At this point you should probably error out since the scp command will probably do something funky.